internal/model/todos.go

```diff
@@ -56,13 +59,15 @@ func CreateTodos(name string, todos []string) ([]Todo, error) {
 	sqlStatement := `INSERT INTO todos(list_id, todo) VALUES($1, $2)
 		RETURNING id, list_id, todo, done`
 	for _, todo := range todos {
-		var createdTodo = Todo{}
+		var createdTodo = &Todo{}
```

Comment:
When you make the struct pointer like this it really doesn't change how you interact with it (which is nice). In other languages if you were dealing with the struct itself you could have a different syntax when accessing its fields than if you were referencing the pointer... Like in C/C++:

```
myStruct.field // non-pointer access
myStructPtr->field // pointer access

```